src/Buildalyzer/Compiler/CompilerLanguage.cs

```diff
@@ -1,17 +0,0 @@
-﻿namespace Buildalyzer;
-
-/// <summary>The compiler language.</summary>
```

Comment:
I disagree with dropping this. I would propose to keep `CompilerLanguage` but convert it to a (Single Value Object) readonly struct. That was not needed yet, but with your proposed solution, it will be.